src/authentication/create-account-method/index.tsx

```diff
@@ -1,8 +1,10 @@
-import React from 'react';
-import { ToggleGroup } from '@zero-tech/zui/components';
+import * as React from 'react';
```

Comment:
to confirm, is this intentional change? Everywhere i see `import React from 'react';` being used

Reply:
Yeah I would have changed this just so I didn't have to explicitly import `FC` for `React.FC` on line 20. 

I probably should have used `React.Component` for a class component rather than functional to stay in line with everything else. My favoured Functional Component approach has crept through here haha. I'll update at another time.